Week-1/cattyboard/index.html

```diff
@@ -0,0 +1,113 @@
+            <img src="https://img.icons8.com/ios-filled/50/000000/settings.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/spotify.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/soundcloud.png" />
+        </div>
+    </nav>
+    <header class="inline">
+
+        <img id="albumimg" src="2Eq--cattyboard.jpg" alt="album image">
+
+        <div class="container">
+
+            <div>
+                <h2>CattyBoard Top 100 Single Charts (11.06.36)</h2>
+                <p>
+                    Unknown Artist <br>
+                    2016. Charts. 100 Songs
+                </p>
+            </div>
+
+            <div class="innerflex">
+
+                <p>
+                    <span> &#9654; Play all </span>
+
+                    <span> &#43; Add to </span>
+
+                    <span> &#8230; More </span>
+                </p>
+            </div>
+        </div>
+    </header>
+    <section class="inline">
+        <table>
+            <tr>
+                <td>1. One Dance</td>
+                <td>Crake feat CatKid & Cyla</td>
+                <td>2:54</td>
+                <td> <span class="syncstyle"> CATTY CLOUD SYNC </span></td>
+            </tr>
+            <tr>
+                <td>2. Panda</td>
+                <td>Cattee</td>
+                <td>4:06</td>
+                <td> <span class="syncstyle"> CATTY CLOUD SYNC </span></td>
+            </tr>
+            <tr>
+                <td>3. Cant Stop the Feeling</td>
+                <td>Catin Cimberlake</td>
+                <td>3:56</td>
+                <td> <span class="syncstyle"> CATTY CLOUD SYNC </span></td>
+            </tr>
+            <tr>
+                <td>4. Work From Home</td>
+                <td>Cat Harmony feat Colla</td>
+                <td>3:34</td>
+                <td> <span class="syncstyle"> CATTY CLOUD SYNC </span></td>
+            </tr>
+        </table>
+    </section>
+    <div class="clearfix"></div>
+    <footer>
+
+        <div class="innerfooterflex">
+            <img id="footeralbumimg" src="2Eq--cattyboard.jpg" alt="album image">
+
+            <div>
+
+                <p>If It Ain't Love</p>
+                <p>Catson Derulo</p>
+            </div>
+        </div>
+
+        <div>
+
+
+            <div id="playcontrols">
+
+                <img style="transform: rotate(180deg);" src="https://img.icons8.com/ios-glyphs/30/ffffff/end--v1.png" />
+                <img src="https://img.icons8.com/ios-glyphs/30/ffffff/pause--v1.png" />
+                <img src="https://img.icons8.com/ios-glyphs/30/ffffff/end--v1.png" />
+                <img src="https://img.icons8.com/ios-glyphs/30/ffffff/repeat.png" />
+                <img src="https://img.icons8.com/ios-glyphs/30/ffffff/replay.png" />
+                <img src="https://img.icons8.com/fluency-systems-filled/48/ffffff/swap-paths.png" />
+            </div>
+        </div>
+
+    </footer>
+</body>
+
+</html>
```

Comment:
lint issues.

Reply:
I'm using Prettier code formatter. It only formats code in readable fashion. But it does NOT remove whitespaces. 

Can you please suggest a VS Code plugin that does this ? I'm not sure what a linter is...